base/algorithms.jl

```diff
@@ -0,0 +1,119 @@
+module Algorithms
```

Comment:
Yes, I forgot to mention that in the OP. It's just that I didn't know to put those functions, but I open to suggestions.